pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.pdm]
+
+[project]
+name = "ocrd-monitor"
+version = "0.1.0"
+description = "A frontend to the OCRD Manager"
+authors = [
+    { name = "Sven Marcus", email = "sven.marcus@tu-braunschweig.de" },
+    { name = "Markus Weigelt", email = "markus.weigelt@slub-dresden.de" },
+    { name = "Robert Sachunsky", email = "robert.sachunsky@slub-dresden.de" },
+]
+dependencies = [
+    "pydantic[dotenv]>=1.10.6",
+    "pydantic[dotenv]",
+    "fastapi>=0.95.0",
+    "Jinja2>=3.1.2",
+    "websockets>=10.4",
+    "uvicorn>=0.19.0",
+    "httpx>=0.23.3",
+]
+requires-python = ">=3.11"
+license = { text = "MIT" }
+[project.optional-dependencies]
+dev = [
+    "beautifulsoup4>=4.11.1",
+    "mypy>=1.1.1",
+    "pytest>=7.2.2",
+    "pytest-asyncio>=0.21.0",
+    "testcontainers>=3.7.1",
+    "black>=23.1.0",
+    "types-requests>=2.28.11.15",
+    "types-beautifulsoup4>=4.12.0.0",
```

Comment:
```suggestion
    "types-beautifulsoup4>=4.12.0.0",
    "types-Pygments>=2.14.0.6",
    "types-ujson>=5.7.0.1",
```

Reply:
Neither `pygments` nor `ujson` are installed in my virtual environment, so those dependencies should not be necessary.

Reply:
Ok then I check this with the CI pipeline whether the error appears still appears here.